cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -104,8 +104,6 @@ spec:
       tolerations:
       - key: CriticalAddonsOnly
```

Comment:
I _think_ we should make this accept everything, with `Exists` only with a key, or have both this and an `Exists`-only thing. @timothysc WDYT?

Reply:
the explanations i found on `CriticalAddonsOnly` are kind of vague. to my understanding these two lines, would help to run the daemonset on a node that already has critical pods.

but this thread suggested that "priority and preemption" (beta in 1.11) handles this "transparently"?
https://github.com/kubernetes/kubernetes/issues/57659#issuecomment-354922646
https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/

Reply:
+1 re make it land everywhere.